packages/hosted-credit-card-integration/project.json

```
@@ -19,5 +19,5 @@
       }
     }
   },
-  "tags": ["scope:integration"]
+  "tags": ["scope:shared"]
```

should we do like this?
```suggestion
  "tags": ["scope:integration", "scope:shared"]
```

---

I thought we were supposed to use only 1 tag 🤔